python/paddle/distributed/auto_parallel/operators/dist_default.py

```diff
@@ -195,7 +197,7 @@ def update_dims_mapping(self, dist_op):
     def forward(ctx, *args, **kwargs):
 
         dist_op_context = ctx.dist_op_context
-        main_block = dist_op_context.get_dst_main_program().global_block()
+        main_block = dist_op_context.get_work_block()
```

Comment:
Please change some names and use `dist_op_context` like these: `dist_op_context.main_block, dist_op_context.startup_block, dist_op_context.src_op, dist_op_context.rank_id`.